src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -211,7 +213,7 @@ private static int doTypeSwitch(Object target, int startIndex, Object[] labels)
      * invocation type is not a method type of first parameter of an enum type,
      * second parameter of type {@code int} and with {@code int} as its return type,
      * or if {@code labels} contains an element that is not of type {@code String} or
-     * {@code Class}.
+     * {@code Class} of the target enum type.
      * @throws Throwable if there is any error linking the call site
      * @jvms 4.4.6 The CONSTANT_NameAndType_info Structure
```

Comment:
Not sure what we do for other bootstraps, but I wonder if the throws clause here should be more specific - for instance the LambdaMetafactory tells when an exception is gonna be triggered. Here it seems like there are a couple of cases which can be called out, where we know that linkage is going to fail.